webapp/src/features/Dashboard/SubscriptionTable.js

```diff
@@ -308,6 +242,20 @@ const SubscriptionTable = () => {
 			)
 		},
 	}
+
+	const handleClick = (e) => {
```

Comment:
this is also a generic name
it should be clear, what are we clicking.

Reply:
updated